ios/Podfile.lock

```diff
@@ -2155,6 +2193,6 @@ SPEC CHECKSUMS:
   SocketRocket: abac6f5de4d4d62d24e11868d7a2f427e0ef940d
   Yoga: 055f92ad73f8c8600a93f0e25ac0b2344c3b07e6
 
-PODFILE CHECKSUM: 79119d2af4f01a2584749dc3926902e66572d896
+PODFILE CHECKSUM: d79a4f70331d49a1a0bbd657163b616621147a8b
 
-COCOAPODS: 1.16.1
+COCOAPODS: 1.15.2
```

Comment:
Update your cocoapods pl

Reply:
Already did it. This is wip and will be fixed with future commits.